poolrpc/trader.swagger.json

```diff
@@ -424,6 +424,88 @@
             }
           }
         },
+        "parameters": [
+          {
+            "name": "start_batch_id",
+            "description": "The unique identifier of the first batch to return, encoded as a compressed\npubkey. This represents the newest/most current batch to fetch. If this is\nempty or a zero batch ID, the most recent finalized batch is used as the\nstarting point to go back from.",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "byte"
+          },
+          {
+            "name": "num_batches_back",
+            "description": "The number of batches to return at most, including the start batch.",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int64"
+          }
+        ],
+        "tags": [
+          "Trader"
+        ]
+      }
+    },
+    "/v1/pool/batch/snapshots/{start_batch_id}": {
```

Comment:
Does this work with `/v1/pool/batch/snapshots/` as well, without `start_batch_id`? It states the parameter is required (line 473), but with or without trailing slash should work the same IMO.

Reply:
Yes, that's what line 409 is for. There are three URL patterns now:
```
/v1/pool/batch/snapshots
/v1/pool/batch/snapshots/{start_batch_id}
/v1/pool/batch/snapshots/{start_batch_id}/{num_batches_back}
```

And you can use the GET parameters too:
```
/v1/pool/batch/snapshots?num_batches_back=10
/v1/pool/batch/snapshots/{start_batch_id}?num_batches_back=10
```

Reply:
My comment was on `/v1/pool/batch/snapshots/`, with a trailing slash and without start_batch_id query param. It looks like that's not included in the pattern. But it's a minor thing.